admin/proc-rhdh-monitoring-logging-aks.adoc

```diff
@@ -0,0 +1,52 @@
+[id='proc-rhdh-monitoring-logging-aks_{context}']
+= Monitoring and logging with Azure Kubernetes Services (AKS) in {product}
+
+Monitoring and logging are integral aspects of managing and maintaining Azure Kubernetes Services (AKS) in {product}. With features like Managed Prometheus Monitoring and Azure Monitor integration, administrators can efficiently monitor resource utilization, diagnose issues, and ensure the reliability of their containerized workloads.
+
+To enable Managed Prometheus Monitoring, use the `-enable-azure-monitor-metrics` option within either the `az aks create` or `az aks update` command, depending on whether you're creating a new cluster or updating an existing one:
+
+[source,bash]
+----
+az aks create/update -g <your-ResourceGroup> -n <your-Cluster> --enable-azure-monitor-metrics
+----
```

Comment:
Allign with the rest of the doc that uses the longer form of the parameter
```suggestion
az aks create/update --resource-group <your-ResourceGroup> --name <your-Cluster> --enable-azure-monitor-metrics
----
```

Reply:
Done